stan/math/rev/mat/fun/cov_exp_quad.hpp

```diff
@@ -120,6 +124,8 @@ class cov_exp_quad_vari : public vari {
  * matrix, along with a pointer to the vari for sigma,
  * and the vari for l.
  *
+ * @deprecated use <code>gp_exp_quad_cov_vari</code>
```

Comment:
Remove comments other than the @depracted line. See above.

Reply:
feel free to just say "same goes for the other files" or something next time...